internal/clients/team4/speaker.go

```diff
@@ -11,15 +11,17 @@ type speaker struct {
 	parent *client
 }
 
-//SpeakerActionPriorities indicate speaker actions in order of priority
-var SpeakerActionPriorities = [...]string{
+//SpeakerActionOrder indicate speaker actions in order of execution
+var SpeakerActionOrder = []string{
 	"SetVotingResult",
 	"SetRuleToVote",
 	"AnnounceVotingResult",
-	"UpdateRules",
 	"AppointNextJudge",
 }
 
+//SpeakerActionPriorities indicate speaker actions in order of priority
+var SpeakerActionPriorities = SpeakerActionOrder
```

Comment:
Why are you doing this? (Sorry, it's late so it might be obvious)

Reply:
It's initialisation - at the beginning, all rules are in play and therefore all rules are high priority

Reply:
Still curious about this? I don't understand why you do this.

Reply:
SpeakerActionOrder is the order these actions are taken according to RunIIGO
SpeakerActionPriority is the order of actions we want our speaker to perform
At the beginning (when all rules are in play) there is no reason to have our own priority of actions
Later when the rules come in/out of play the SpeakerActionPriority gets updated
SpeakerActionOrder is always fixed as mirroring the orchestration.go execution order

Reply:
Global variable

Reply:
SpeakerActionPriorities = make([]string, len(SpeakerActionOrder ))
	copy (SpeakerActionPriorities , SpeakerActionOrder )